RibonDAO/app
# [PROD3-185] Cancel subscription

# Description

Page to list all monthly contributions
Modal to send e-mail to cancel contribution

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#PROD3-185]()

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Verify if when you made a contribution, get redirect to monthly contribution and the arrow to back redirect to promoters screen
- Verify if you receive the e-mail
- Verify if the contribution has canceled when click the email
